serverless/sql-databases/reference-content/planned-maintenance.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: Maintenance process for Serverless SQL Databases
+  description: This page explains the maintenance process for Serverless SQL Databases and its potential impact on your applications
+content:
+  h1: Maintenance process for Serverless SQL Databases
+  paragraph: This page explains the maintenance process for Serverless SQL Databases and its potential impact on your applications
+tag: serverless sql databases maintenance service interruption planned operation
+dates:
+  validation: 2024-07-09
+  posted: 2024-07-09
+---
+
+## Overview
+
+To add features, deploy security fixes or corrective patches, we perform regular maintenance operations. **Most maintenance operations are transparent to you**, and have no impact on your database availability, or performance.
+
+However, **some maintenance operations may impact your database availability** or require actions from your side. In this case, **we will inform you beforehand**, so you can **follow the recommendations below** to ensure the impact on your application is minimal.
+
+The maintenance operations particularly concern our [connection pooler](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) for now, as we are working to make it as transparent as possible in future Serverless SQL Databases updates.
+
+## Impact on your application and recommendations
+
+A maintenance operation impacting your SQL client connections to the database will usually:
+
+- Disconnect your SQL connections to the database **during 10 seconds**
```

Comment:
```suggestion
- Disconnect your SQL connections to the database **during less than 1 minute**
```
After technical discussion, even if the operation should take 10 seconds in most cases, mentioning 1 minute maximum delay is more cautious